src/App.vue

```diff
@@ -71,6 +71,8 @@ const contactmomentStore = useContactmomentStore();
   --header-height: 6rem;
   --text-margin: 1.5rem;
 
+  --line-height-default: 1.5;
```

Comment:
Eliminating the need to set it in each individual case.

(There's a normalise css included which sets it to `1.0`, decided not to override that)